coda.io.website.json

```diff
@@ -0,0 +1,31 @@
+{  
+    "providerId":"coda.io",
+    "providerName":"Coda",
+    "serviceId":"website",
+    "serviceName":"Coda Site",
+    "version": 1,
+    "syncPubKeyDomain" : "coda.io",
+    "logoUrl":"https://cdn.coda.io/icons/png/color/coda-192.png",
+    "description":"Allows serving Coda documents from your own domain",
+    "variableDescription":"`host` refers to the subdomain that the user wishes to use (not just limited to 'www'). `subdomain` refers to the subdomain for Coda's custom domain server. `docId` refers to the coda document used for custom domain. `verification` refers to the TXT host for doc ownership verification, differs based on A or CNAME. `ipAddress` refers to the proper IP for an apex domain to point to.",
+    "records":[  
+       {  
+          "type":"CNAME",
+          "host":"%host%",
+          "pointsTo":"%subdomain%.coda.io",
+          "ttl":3600
+       },
+       {
+           "type": "TXT",
+           "host": "%verification%",
+           "data": "Doc:%docId%",
+           "ttl":3600
+       },
+       {
+           "type": "A",
+           "host": "@",
+           "pointsTo": "%ipAddress",
```

Comment:
Seems like if we want to use three different IP addresses, we may need three variables here? I see other templates doing that sort of thing and I don't see any documentation about variables allowing multiple IP addresses

Reply:
Yes, 3 A records with 3 IP variables would be needed, if the IPs are variable all the times.